Tests.Integration.Presentation.Web/Tools/EntityOptionHelper.cs

```diff
@@ -105,5 +107,15 @@ public static async Task<HttpResponseMessage> SendChangeOptionIsObligatoryAsync(
             Assert.Equal(HttpStatusCode.NoContent, response.StatusCode);
             return response;
         }
+
+        public static async Task<List<OptionDTO>> GetOptionsAsync(string resource, Cookie optionalLogin = null)
```

Comment:
I had to add a new method in order to get the options, because the V2 Helper method required an organizationUuid, so I thought it's unnecessary to create an organization just to get the options